src/Lecture1.hs

```diff
@@ -77,7 +82,21 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z = (max (max x y) z) - (min (min x y) z)
+    where 
+        max :: Int -> Int -> Int
+        max a b = 
```

Comment:
You don't need to implement `max` and `min` functions. These are already standard functions 👍🏻

Reply:
Replaced